lib/Server.js

```diff
@@ -546,6 +544,9 @@ class Server {
   }
 
   createServer() {
+    const https = require('https');
+    const http = require('http');
```

Comment:
I'm not sure it'd make a difference, but I think we can move these below just like `require('spdy')`.

Reply:
Feel free to send PR